test/test_public_bindings.py

```diff
@@ -33,6 +33,7 @@ def test_no_new_bindings(self):
         torch_C_allowlist_superset = {
             "AggregationType",
             "AliasDb",
+            "AliasInfo"
```

Comment:
Do we really want `torch.AliasInfo` to be a thing? Is this new public API properly documented?

Reply:
changed it to torch._C._AliasInfo so this won't be an issue